examples/search_bar/main.rs

```diff
@@ -19,16 +20,21 @@ fn build_ui(application: &gtk::Application) {
     window.set_child(Some(&search_bar));
     search_bar.set_key_capture_widget(Some(&window));
 
-    let search_box = gtk::Box::new(gtk::Orientation::Horizontal, 6);
+    let search_box = gtk::Box::new(gtk::Orientation::Vertical, 6);
     search_bar.set_child(Some(&search_box));
 
     let entry = gtk::SearchEntry::new();
     entry.set_hexpand(true);
     search_box.append(&entry);
     search_bar.connect_entry(&entry);
 
-    let menu_button = gtk::MenuButton::new();
-    search_box.append(&menu_button);
+    let label = gtk::Label::new(None);
+    label.set_hexpand(true);
+    search_box.append(&label);
+
+    entry.connect_search_changed(clone!(@strong entry => move |_| {
```

Comment:
```suggestion
    entry.connect_search_changed(clone!(@weak entry => move |_| {
```
don't use strong references in general :)